db/migrations/20220923181228-create-orders-table.js

```diff
@@ -1,37 +1,46 @@
 module.exports = {
   up: async (queryInterface, Sequelize) => {
-    await queryInterface.createTable('orders', {
+    await queryInterface.createTable("orders", {
       id: {
         allowNull: false,
         autoIncrement: true,
         primaryKey: true,
         type: Sequelize.INTEGER,
       },
       order_details: {
+        allowNull: false,
         type: Sequelize.JSON,
       },
       customer_id: {
+        allowNull: false,
         type: Sequelize.INTEGER,
+        references: {
+          model: "users",
+          key: "id",
+        },
       },
       amount: {
+        allowNull: false,
         type: Sequelize.FLOAT,
       },
       status: {
+        allowNull: false,
```

Comment:
perhaps it might be useful to add comments about what the expected `status` values are - e.g. confirmed, inprog, shipped out, in transit, etc.